### PR TITLE
Ensure ReDoc doesn't throw warnings

### PR DESCRIPTION
**Pull request by Leon99**
https://github.com/mattfrear/Swashbuckle.AspNetCore.Filters/issues/69

### CODE REVIEW

File: src/Swashbuckle.AspNetCore.Filters/Examples/RequestExample.cs [hunk 1]
```diff
@@ -65,6 +64,10 @@ internal class RequestExample
                     definitionToUpdate.Example = formattedExample;
                 }
             }
+            else
+            {
+                bodyParameter.Schema.Example = formattedExample; // set example on the paths/parameters/schema/example property
```
[the remark this answers is not in the document]
Comment (mattfrear) [reply src/Swashbuckle.AspNetCore.Filters/Examples/RequestExample.cs]: Actually, I just had a look and that line has coverage already, so nothing needed. I will merge to master.

**Comment by mattfrear** [comment]
Thanks for the PR. I'll include it in the next release, probably later today.

**Comment by Leon99** [comment]
Great, thank you

**Comment by mattfrear** [comment]
Released https://www.nuget.org/packages/Swashbuckle.AspNetCore.Filters/4.5.2

**Pull request merged**